general/clients/web-config.md

```diff
@@ -23,6 +23,26 @@ Overriding the default `config.json` can be done with an additional volume param
 
 We would recommend obtaining the [latest copy of the file](https://github.com/jellyfin/jellyfin-web/blob/master/src/config.json) to pre-populate your configuration directory before starting Jellyfin for the first time; unlike most other components of this directory, it will not be created automatically on first run.
 
+## Customizations
+
+### Custom Menu Links
+
+Jellyfin 10.8 adds the ability to specify custom links to be inserted in the navigation menu via the `config.json` file. Links are configured with a `name`, `url`, and optional `icon` properties. The icon is specified using the name of an icon from the [Material Design Iconfont](https://jossef.github.io/material-design-icons-iconfont/) used in Jellyfin Web. By default the "link" icon will be used.
```

Comment:
```suggestion
Jellyfin 10.8 adds the ability to specify custom links to be inserted in the navigation menu via the `config.json` file. Links are configured with a `name`, `url`, and optional `icon` property. The icon is specified using the name of an icon from the [Material Design Icons font](https://jossef.github.io/material-design-icons-iconfont/) used in Jellyfin Web. By default the "link" icon will be used.
```

I think that's a bit nicer to read.
Might also want to insert line-breaks between the sentences (which Markdown ignores) for easier git handling.